quickstart/101-azurerm_dns_zone/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+resource "random_string" "name" {
+  length  = 13
+  lower   = true
+  numeric = false
+  special = false
+  upper   = false
+}
+
+resource "azurerm_dns_zone" "zone" {
+  name                = "www.${random_string.name.result}.azurequickstart.org"
```

Comment:
I would think we would want this to be a variable as well instead of randomly generating a dns zone name.??

Reply:
A random string is necessary for automating test since there's no user input in pipeline, and we might run multiple tests concurrently so it cannot be a static hard coded string, I think we can add a new variable in case a user want to set one, if the variable's value is `null` then we use random string instead, WDYT?

Reply:
That makes sense to me.

Reply:
Done.